e2e/tests/components/select/forms/select-forms.e2e-spec.ts

```diff
@@ -589,4 +589,21 @@ describe('Select (forms) Tests', () => {
 
     });
 
-});
\ No newline at end of file
+    it('should correctly validate the tag input values', async () => {
+        await page.clickOnCheckbox(page.checkboxMulti);
+        await page.clickOnDropdown(true);
+
+        await page.clickOnCountry(true, 0);
+        expect(await page.confirmClassExists(page.dropdown, 'ng-invalid')).toBe(true);
```

Comment:
Could be good to add a description to this assertion, it can help with interpreting test failures.